src/profiler.cpp

```diff
@@ -1647,6 +1652,104 @@ void Profiler::dumpText(Writer& out, Arguments& args) {
     }
 }
 
+/*
+ * If the value has been seen already, return its idx in `values`. If not, update the map,
+ * add the value to `values`, and return the index.
+ */
+static inline u64 getOrSetIndex(
+        std::unordered_map<std::string, u64>& idx_map,
```

Comment:
u32? It's unrealistic to have more than 4B elements.

Reply:
3fbdc061c5d4a003bf521a70cf3a7ca332c1fe47